recipe/meta.yaml

```diff
@@ -27,11 +28,14 @@ requirements:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
   host:
-    - sqlite >=3.11
-    - libtiff
-    - libcurl
+    - sqlite {{ sqlite }}
+    - libtiff {{ libtiff }}
+    - libcurl 7.88.1
+    - nlohmann_json 3.11.2
+    - gtest 1.14.0 # [unix]
```

Comment:
Why not use our gtest on Windows too?

Reply:
It lead to a test build failure. I could investigate on a dev instance but did not take time to start one.

Reply:
PROJ tests from docs https://proj.org/en/9.3/install.html#id7